lib/private/Authentication/Exceptions/ExpiredTokenException.php

```diff
@@ -27,17 +27,19 @@
 
 use OC\Authentication\Token\IToken;
 
-class ExpiredTokenException extends InvalidTokenException {
-	/** @var IToken */
-	private $token;
-
-	public function __construct(IToken $token) {
-		parent::__construct();
-
-		$this->token = $token;
+/**
+ * @deprecated 28.0.0 use OCP version instead
```

Comment:
```suggestion
 * @deprecated 28.0.0 use {@see \OCP\Authentication\Exceptions\ExpiredTokenException} instead
```

That is clickable from phpstorm at least :)